### PR TITLE
Cliar uat family avg

### PR DESCRIPTION
**Pull request by galileukim**
@kannanv93, please review the changes I have introduced to our CLIAR_UAT. you will find that all the lingering issues we had regarding the ETL have been addressed, including (a) the partial benchmarking and (b) the revised set of indicators.

based on this new data ETL, the app deploys (at least locally). the issues I have seen are the following:

1. the variables list is not adapting dynamically, leading to frustrating experiences selecting an institutional family average that does not exit.
2. I have noted that some maps are not compiling correctly, in particular for anti-corruption. could we please check that?
3. I have switched our use of compiled_indicators from `compiled_indicators_updated.csv` to `compiled_indicators.csv`. could you please verify this is working as intended?

### CODE REVIEW

**Comment by galileukim** [comment]
@kannanv93 please note that all the changes necessary for the CLIAR_UAT_family_avg have been finalized. I see one conflict regarding vars-control.R that I defer to you on how to resolve. Once that is solved, we should be able to merge this branch into `CLIAR_UAT`.

**Review by kannanv93 — APPROVED** [review]
Changes Approved

**Pull request merged**
